frontend/src/libraries/language.ts

```diff
@@ -5,68 +5,17 @@
  * - https://github.com/near/near-wallet/blob/master/src/utils/getUserLocale.js
```

Comment:
Please review this file changed thoroughly as well.
I don't see a reason for us to have browser-based language detection, as we'll definitely will go through `App.getInitialProps` where we get the same data (`Accept-Language` instead of `navigator.language`, `req.headers.cookie` instead of `window.cookie`) and will set the cookie for the user.

Reply:
> I don't see a reason for us to have browser-based language detection

Well,

1. Probably that was inherited from Wallet, which is an SPA application - we received i18n contribution from folks who also worked on i18n for Wallet
2. If we even consider to have a fallback from SSR to SPA, we would need to have browser-based language detection (yet, we may address it at that point, so we don't need to make any decisions right now)

Overall, it looks good to me so far